src/app/pages/ArticlePage/PagePromoSections/RelatedContentSection/index.jsx

```diff
@@ -52,11 +63,13 @@ const renderRelatedContentList = (item, index) => {
 const RelatedContentSection = ({ content }) => {
   const { translations, script, service } = useContext(ServiceContext);
 
-  if (!pathEq(['type'], 'relatedContent', content)) return null;
+  const filteredContent = filterContent(content);
```

Comment:
Naming convention is a little confusing here but I don't really know what else you could name these :(

Reply:
Yea tricky one. Renamed the function and variable name to something a little more clear, hopefully!